test-cypress/stubs/graphql/graphql-stubs.js

```diff
@@ -1,15 +1,8 @@
-export const ENDPOINT_COUNTRIES = '/rest/repositories/test/graphql/countries';
-export const ENDPOINT_RICKMORTY = '/rest/repositories/test/graphql/rickmorty';
-
 export class GraphqlStubs {
-    static stubGetEndpoints(repositoryId, fixture = 'graphql-endpoints.json', delay = 0) {
-        const fuxturePath = `/graphql/endpoints/${fixture}`;
-        cy.intercept('GET', `/rest/repositories/${repositoryId}/graphql/endpoints`, {
-            fixture: fuxturePath,
-            statusCode: 200,
-            delay: delay
-        }).as('get-endpoints');
-    }
+    static spyExportEndpointDefinition(repositoryId, endpointId) {
+        cy.intercept('GET', `/rest/repositories/${repositoryId}/graphql/manage/endpoints/${endpointId}/export`)
+            .as('export-endpoint-definition');
```

Comment:
Just a thought, but we can return the alias and then use it in the `cy.wait` instead of knowing it or having to look.

Reply:
yeah, it'd be nice. Will consider it in future.